### ISSUE TITLE
Why Does my wxpython program auto tranlated into other langauge when my locale lanauge is set to en-US?

### ISSUE THREAD

**Issue by wekay102200** [wxWidgets/Phoenix#729]
**Operating system**:
linux ubuntu 14.04
**wxPython version**:
3.0.2
**Stock or custom build**:
**Python version**:
2.7
**Stock or custom build**:

**Description of the problem**:
When I use self.my_locale = wx.Locale(LANGUAGE_ENGLISH_US) to set my progream in en_US enviroment.
but some fo text in menu items such as New,Close ,Open,etc.
such text is auto translated into Chinese.
I think this is a operation system level problem.
How to avoid such auto tranlated problem?

```python
# Put code sample here
```

**Comment by RobinD42** [comment]
Do you use the stock IDs for those menu items? (`wx.ID_NEW`, `wx.ID_OPEN`, etc.) If so then the text for those items comes from the system.

**Comment by wekay102200** [comment]
@RobinD42 Yes, I have Use stodkc ID suc as wx.ID_NEW, wx.ID_OPEN etc.
Do you mean these item of id will come from my local system?
Can You tell me why?
I found the  auto tranlated problem only happen on ubuntu,but on centos It will not happen.
What's the reason may be?

**Comment by wekay102200** [comment]
@RobinD42 Yes, I have Use stodkc ID such as wx.ID_NEW, wx.ID_OPEN etc.
Do you mean these item text of id will come from my local os system?
Can You tell me why?
I found the auto tranlated problem only happen on ubuntu,but on centos It will not happen.
What's the reason may be?

**Comment by wekay102200** [comment]
How to prevent  auto tranlated whe I use stock IDs? @RobinD42

**Comment by mesalu** [comment]
The different behavior between ubuntu and centos may be caused by different system locales.
You can check each installations locale settings with the `locale` command in a shell.

**Comment by RobinD42** [comment]
> Do you mean these item text of id will come from my local os system?
> Can You tell me why?

Yes. This is one of the ways that wxWidgets helps you to create cross-platform applications that conform to the standards of the native APIs.  The whole concept of "stock" is that it is normalized and consistent and compliant to standards.

> How to prevent auto tranlated whe I use stock IDs?

I'm not sure that you can. When a stock ID is used then the native gtk menu item is created with a different gtk API than when using non-stock IDs.

**Comment by RobinD42** [comment]
> The different behavior between ubuntu and centos may be caused by different system locales.

It's also possible that the appropriate language catalogs for the target language are not installed.
